notifier/webhook.go

```diff
@@ -74,14 +75,39 @@ func (s *Webhook) webhookURL() (string, error) {
 	return s.buildWebhookURL(url)
 }
 
-func (s *Webhook) notify(title string, message string) error {
+func (s *Webhook) webhookStatusURLs() (string, string, error) {
+	successURL := s.viper.GetString("success.url")
+	failureURL := s.viper.GetString("failure.url")
+
+	if s.buildWebhookStatusURLs == nil {
+		return successURL, failureURL, nil
+	}
+
+	return s.buildWebhookStatusURLs(successURL, failureURL)
+}
+
+func (s *Webhook) notify(title string, message string, notifyType ...int) error {
 	logger := s.getLogger()
 
+	successURL, failureURL, err := s.webhookStatusURLs()
+	if err != nil {
+		return err
+	}
+
 	url, err := s.webhookURL()
 	if err != nil {
 		return err
 	}
 
+	if len(notifyType) > 0 {
+		if len(successURL) > 0 && notifyType[0] == notifyTypeSuccess {
```

Comment:
Why not just let `notifyType` is an int, this is an array but, just have one member?

Reply:
In the previous version there was no notifyType input at all, so I decided to use it as a variadic input parameter. This way we can ensure compatibility with the old code and provide or not provide this argument. 
But yes, as you mentioned, it will be treated as an array then.